bliss/surveys/dc2.py

```diff
@@ -320,7 +320,6 @@ def read_image_for_bands(self, image_index):
                 self.image_lim[0], self.image_lim[1]
             )
 
-            image += bg
             image_list.append(image)
             bg_list.append(bg)
```

Comment:
Perhaps we don't need to record `bg` at all any more?

Reply:
waiting on incoming dc2 change (in the open yd PR) that will do the same